src/pool/srv_pool.c

```diff
@@ -6481,112 +6459,6 @@ transfer_ranks_buf(d_rank_t *ranks_buf, size_t nranks,
 	return rc;
 }
 
-/**
- * Send CaRT RPC to pool svc to get list of storage server ranks.
- *
- * \param[in]	uuid		UUID of the pool
- * \param[in]	svc_ranks	Pool service replicas
- * \param[out]	ranks		Storage server ranks (allocated, caller-freed)
- *
- * return	0		Success
- *
- */
-int
-ds_pool_svc_ranks_get(uuid_t uuid, d_rank_list_t *svc_ranks,
```

Comment:
While I am all for removing unused code (I derive a kind of glee from taking a chainsaw to codebases), I do wonder if we should leave this battle-tested function, in case we have a future use case for it.